test/explorer/swagger-explorer.spec.ts

```diff
@@ -832,11 +832,8 @@ describe('SwaggerExplorer', () => {
           name: 'page',
           required: true,
           schema: {
-            type: 'array',
-            items: {
-              type: 'string',
-              enum: ['d', 'e', 'f']
-            }
+            enum: ['d', 'e', 'f'],
+            type: 'string'
```

Comment:
Why did you remove this test? Can you please add a new one instead?

Reply:
I have added a new test. Let me know if it needs adjustments.